src/rdsk.c

```diff
@@ -266,9 +252,9 @@ int scandir_filter_dm(const struct dirent *list) {
  */
 struct RC_PROFILE *search_cache_targets(char *return_message)
 {
-	int num, num2, num3, n = 0, i, z;
-	struct dirent **list, **nodes, **maps;
-	char file[NAMELEN] = {0};
+	int num, num2, n, i;
+	struct dirent **list, **maps;
+	char file[NAMELEN] = {0}, link[0xf] = {0};
```

Comment:
`link` size is small, maybe a bigger value can protect us from nasty problems in the future.

Reply:
Good point.